src/htdocs/monitoring/operations/station.php

```diff
@@ -0,0 +1,14 @@
+<?php
+  include_once 'functions.inc.php';
+  include_once __DIR__ . '/conf/config.inc.php';
+
+  $network = param('network');
+  $station = param('station');
+
+  if ($network != null && $station != null &&
+      preg_match('/^[a-zA-Z]+$/', $network) &&
+      preg_match('/^[a-zA-Z]+$/', $station)) {
```

Comment:
station codes may also include numbers, wouldn't hurt to add for networks just in case